wgsl/index.bs

```diff
@@ -880,30 +880,35 @@ shader that goes beyond the specified limits.
   <thead>
     <tr><th>Limit<th>Minimum supported value
   </thead>
-    <tr><td>Maximum number of members in a [=structure=] type<td>16383
-    <tr><td>Maximum [=nesting depth=] of a [=composite=] type<td>255
+    <tr><td>Maximum number of members in a [=structure=] type<td>1024
```

Comment:
All the limits were previously 2^N - 1, we should probably stick with that.

Reply:
Done.

I left the variable size ones as `2^N` to be consistent with the workgroup storage size limit and also because `2^N - 1` isn't a size that you can actually use since we have no 1-byte types.